lib/membrane_http_adaptive_stream/storages/file_storage.ex

```diff
@@ -15,6 +15,18 @@ defmodule Membrane.HTTPAdaptiveStream.Storages.FileStorage do
   @impl true
   def init(%__MODULE__{} = config), do: config
 
+  @impl true
+  def store(
+        _parent_id,
+        _name,
+        _contents,
+        _metadata,
+        %{mode: :binary, type: :partial_segment},
+        state
+      ) do
+    {:ok, state}
+  end
```

Comment:
So the file storage doesn't support partial segments?

Reply:
But the partial segments will be present in the playlist, won't it break anything? Doesn't it make sense to store them and remove them when the full segment arrives?